### ISSUE TITLE
Cannot get import of custom js file to work

### ISSUE THREAD

**Issue by tweidema** [riok/Kreya#259]
**Describe the bug**
Trying to create a custom js file as explained here: https://kreya.app/docs/scripting-and-tests/#sharing-code, but Kreya cannot find the file

**To Reproduce**
I have created this simple file called shared.js in my project directory (same directory that the .krproc file resides in).
```
thw@Snow:~/kreya/buapi_attribute_poc$ cat shared.js
export function logit() {
    kreya.trace("Called from module");
}
```
I try to import from the script section of a gRPC call with
```
import { logit } from "shared.js";
```
However the editor shows a red line under "shared.js" with a pop-up text of "cannot find module 'shared.js'. Did you mean to set the 'moduleresolution' option to 'node' or to add aliases to the 'paths' option? (2792)"

**Expected behavior**
It should be able to find my shared.js file.

**Environment (if possible, copy the information from the error dialog or the About menu):**
 - OS: Ubuntu 20.04.6 LTS
 - Kreya Version 1.15.0

**Comment by CommonGuy** [comment]
I just tried this and it seems like this is a bug on the editor. The import of the script does work, but it isn't shown correctly.

If you change the script in the gRPC call to 
```
import { logit } from "shared.js";

logit();
```

then you should see the log message in the Trace tab.

**Comment by tweidema** [comment]
Hmm, it works partly like you say, but something is strange. If I have the following in the script-tab:
```
import { logit } from "shared.js";
logit()
kreya.trace("from script-tab")

kreyaGrpc.onResponse ( msg => {
   logit()
})
```
and I run it as a single request, by clicking the "Send" button, I see all the loglines in trace:
```
[06:43:01.955] ScriptMessage
               message: Called from module 

[06:43:01.955] ScriptMessage
               message: from script-tab 

[06:43:01.956] KreyaCallStart
               disableServerCertificateValidation: true 
               mode: Grpc 
...
[06:43:02.029] ScriptMessage
               message: Called from module 

[06:43:02.030] KreyaCallStop
```
but, if I run the same operation from a collection, I only get the last log, the one from logit() inside the onResponse handler, not the first 2 logs.

**Comment by CommonGuy** [comment]
Strange, I am unable to reproduce this:
![grafik](https://github.com/user-attachments/assets/f27552ee-8d5e-4b55-afa5-8a72ffb27664)

Are you able to reproduce this consistently?

**Comment by tweidema** [comment]
Yes, but it's even weirder. Here are some details I did not notice when I first reported it.
If I run a collection, then left-click on the first operation twice (first time shows the collection run with "ok"'s) and go to the trace tab the logs are shown. The first 3 are:
![from_collection](https://github.com/user-attachments/assets/c8261248-d361-4816-9109-ad2b8cd22ef9)
If instead I right-click on the first operation (from the collection) and select "Open operation", and go to trace tab, the first 3 are:
![from_operation](https://github.com/user-attachments/assets/b084fafe-39ea-4a67-8693-4cc7e5405c17)
Note the timestamps, it is the same run. But when viewing from operation the first 2 lines are missing.

**Comment by CommonGuy** [comment]
This only happened when the you run a collection and the operation is open at the same time. Very early Trace messages were not displayed in that case. This will be fixed with the next release

**Comment by CommonGuy** [comment]
Included in Kreya 1.16, which was just released
